### ISSUE TITLE
Project still alive?

### ISSUE THREAD

**Issue by uwolfer** [Leonti/CSSInliner#1]
@Leonti: Do you still maintain this project?

We have fixed some things and added some features: https://github.com/tocco/CSSInliner

**Comment by Leonti** [comment]
@uwolfer: yes, I don't have much time at the moment to write new code for it, but if you could create a pull request with your changes I would be grateful.
Thanks!
